daemons/pacemakerd/pacemaker.sysconfig

```diff
@@ -81,6 +81,14 @@
 # host reboot. The default is unset.
 # PCMK_panic_action=crash
 
+# If a node is notified that it has been fenced, either fencing was
+# misconfigured, or fabric fencing that doesn't sever cluster communcation was
+# used. This specifies what the node should do in response. The default "panic"
+# will reboot the local node if possible, whereas "stop" will stop pacemaker if
+# possible and leave it stopped. There is no guarantee a node will receive
+# notification of its own fencing, regardless of how fencing is configured.
+# PCMK_fence_reaction=panic
```

Comment:
Looking at the fix, fence_reaction appears to be what you set with cluster_property_set.
Do you need this environment variable?

Reply:
Whoops, that was an earlier attempt. Thanks for catching that, I'll get rid of it.